### ISSUE TITLE
List is displayed in EN, needs to be translated to AR!

### ISSUE THREAD

**Issue by adhamgsalem** [talhakpg/TeleTashafi#62]
![‎‫تشافي للرعاية الطبية‬‎ 2021-07-22 15-39-02](https://user-images.githubusercontent.com/7084124/126669465-0886cdbc-adef-4959-9eb3-4de0ae5479b8.png)

**Comment by Mahinder-Singh** [comment]
Please provide all list data in Arabic as well

**Comment by Mahinder-Singh** [comment]
Dropdown List related point will be covered and linked in Issue #35 for Arabic  #34 for English please provide data on this

**Comment by hassanahak** [comment]
Yes Mahinder, the QA tester are testing systematically. Most likely will not be able to collect all the drop down list . If you kindly provide me with all drop down list, I will translate them.

Please include drop drwn list in registration as well.
